raybot/commands/lunchshame.rb

```diff
@@ -33,7 +35,7 @@ class LunchShame < SlackRubyBot::Commands::Base
           end
         end
 
-        client.say(channel: "C1TUV5XFA", text: picker + " is picking up today.")
+        client.say(channel: "C1TUV5XFA", text: picker + " is picking up " + restaurant_name + "today.")
```

Comment:
Not your fault, but it seems like we should only print this is meal_today is `true`?

Reply:
fixed